Unit-01/04-flask-crud/app.py

```diff
@@ -1,3 +1,44 @@
-from flask import Flask
+from flask import Flask, render_template, redirect, url_for, request
+from flask_modus import Modus
+from snack import Snack
 
-snack_list = []
\ No newline at end of file
+app = Flask(__name__)
+modus = Modus(app)
+
+snack_list = []
+
+@app.route('/snacks', methods=["GET", "POST"])
+def index():
+	if request.method == "POST":
+		snack_list.append(Snack(request.form.get('name'),request.form.get('kind')))
+		return redirect(url_for('index'))
+	return render_template('index.html', snack_list=snack_list)
+
+@app.route('/snacks/new')
+def new():
+	return render_template('new.html')
+
+@app.route('/snacks/<int:id>', methods=["GET", "PATCH", "DELETE"])
+def show(id):
+	found_snack = [snack for snack in  snack_list if snack.id == id][0]
```

Comment:
This all looks good. You may want to consider handling the case where the id isn't valid - right now, your server will probably crash with an `IndexError`